### DISCUSSION TITLE
funny microsoft teams thing (exploit)

### DISCUSSION THREAD

**Discussion by jazzymakesstuff** [3kh0/ext-remover#1416] [Exploits JOIN TITANIUM NETWORK TO SHARE PLEASE]
some kids at my school figured this out awhile ago, and it got blocked just because of our tomfoolery. but this will probably work for you guys.

im citing this from an old entry i made on my site.

1. make sure microsoft teams is on your apps bar. if it isn't, install it in the chrome store. if you for some reason can't do this, cry in a corner.
2. click on it and make a new account using random characters until you get a string of characters that hasnt been resgistered yet.
3. review the terms of service and fiddle around with different things until you get the option to sign in with github.
4. click the github icon after clicking sign in with github.
5. once you are redirected to github, search for google.
6. there should be a buisness link or something to the right that directs you to google.com.

now let me tell you about some bullshit called the secondary rate limit:

1. if you go on github enough via this, you will receive a secondary rate limit. luckily, there is a way to mitigate this.
2. click on the little thing that says @GithubStatus, or any other elon musk site thing that may happen to appear.
3. on twitter, click sign in with apple.
4. click on the apple logo (no shit)
5. search up youtube in the apple search bar.
6. scroll down and click "visit developer site".
7. you can stop here if you're a nerd, or you can click on the youtube settings/widget bar and click on the terms of service.
8. click on the google license link at the bottom.
9. yippie yay! you did it!

there is probably a way to simplify the latter part, but my monkey brain doesn't think that far ahead.

**Comment by py660** [comment]
Pretty sure the first part was discovered before (either called a webview exploit or something else)

The second part might be useful, but I would recommend creating a pull request and amending the repo.

**Comment by Luigi-Master** [comment]
Awesome exploit, If I were you, I would ask them to put this on the repo.

**Comment by catfoolyou** [comment]
Yet another webview exploit
